examples/matmul.py

```diff
@@ -27,6 +28,15 @@ def check(m: int, k: int, n: int) -> None:
     run_example(kernel_with_bias, expected_with_bias, (x, y))
 
 
+def matmul_tritonbench(
+    a: torch.Tensor, b: torch.Tensor, bias: torch.Tensor | None
+) -> Callable:
+    """Wrapper for tritonbench that matches its interface."""
+    if bias is not None:
+        return lambda: matmul(a, b, lambda acc, tile: acc + bias)
```

Comment:
Don't you need to read a tile of the bias?  Is this tested?

Reply:
sorry it was not tested - updated to `bias[tile[1]]` and tested to confirm